test/net/sourceforge/kolmafia/session/ChoiceControlTest.java

```diff
@@ -504,4 +504,37 @@ void handlesExtracting() {
       }
     }
   }
+
+  @Nested
+  class WereProfessorResearch {
+    @ParameterizedTest
+    @CsvSource({"2, test_choice_wereprofessor_no_liver_two_stomach.html"})
```

Comment:
I don't think there's much point having a parameterized test with exactly one set of parameters.